course_discovery/apps/api/v1/views/courses.py

```diff
@@ -415,6 +417,12 @@ def update_course(self, data, partial=False):  # pylint: disable=too-many-statem
         return_dict.update(serializer.data)
         return Response(return_dict)
 
+    def _is_course_run_reviewed(self, course):
+        """ Checks if any course run for a course is being reviewed """
+        if course.course_runs.filter(status=CourseRunStatus.Reviewed).exists():
+            return True
+        return False
```

Comment:
No need to return boolean explicitly.
```suggestion
        return course.course_runs.filter(status=CourseRunStatus.Reviewed).exists():
```